src/fparser/common/tests/test_readfortran.py

```diff
@@ -340,6 +340,258 @@ def test_base_free_continuation(log):
     assert result == expected
 
 
+def check_include_works(fortran_filename, fortran_code, include_info,
+                        expected, tmpdir, ignore_comments=True):
+    '''Utility function used by a number of tests to check that a include
```

Comment:
s/a include/include/ and please document the interface to this routine.

Reply:
Fixed typo. You want me to document the interface of a test?

Reply:
A utility routine inside a test - yes!

Reply:
Boo, I don't remember agreeing that :-)

Reply:
Done, grr.